### PR TITLE
Modify oem-qa-checkbox-installer and TFG config

### PR DESCRIPTION
**Pull request by seankingyang**
Correct the WIFI SSID from l3-01 to l4 and add WIFI BE:
- `oem-qa-checkbox-installer/conf/plainbox.conf`
- `testflinger_yaml_generator/template/launcher_config/env.conf`

Add WWAN config
- `testflinger_yaml_generator/template/launcher_config/env.conf`

### CODE REVIEW

**Review by hanhsuan — APPROVED** [review]
LGTM

**Review by scotthu27 — APPROVED** [review]
LGTM+1

File: Tools/PC/testflinger_yaml_generator/template/launcher_config/env.conf [hunk 1]
```diff
@@ -28,3 +33,6 @@ STRESS_BOOT_WAKEUP_DELAY = 60
 STRESS_BOOT_WAIT_DELAY = 60
 STRESS_SUSPEND_SLEEP_THRESHOLD = 10
 STRESS_SUSPEND_RESUME_THRESHOLD = 5
+WWAN_APN = internet
+WWAN_CONTROL_IF = wwan0mbim0
```
Comment (stanley31huang) [inline Tools/PC/testflinger_yaml_generator/template/launcher_config/env.conf:37]: is this value should be `mbim0`?

**Review by stanley31huang — APPROVED** [review]
LGTM

**Pull request merged**